pages/documentation/grid/model/index.md

```diff
@@ -10,150 +10,122 @@ latex: true
 
 ## Introduction
 
-In this page the different network components are described in terms of electrotechnical representation.
-Each component is identified through a unique ID, and optionally by a name that is easier to interpret for a human.
-Note that the equipments in the IIDM model may be flagged as fictitious, in order to fine tune the network modelling.
+PowSyBl uses an internal grid model initially developed under the iTesla project, a research project funded by the European Union 7th Framework programme (FP7). The grid model is known as iIDM (iTesla Internal Data Model). One of the iTesla outputs was a toolbox designed to support the decision-making process of power system operation from two-days ahead to real time. The iIDM grid model was at the center of the toolbox.
```

Comment:
I am not sure to get the rules for using _italic_ and **bold** in that descritpion. For example, we can find _node/breaker_, **node/breaker** and node/breaker. I am a bit lost, maybe _italic_ should be removed. @mathbagu do we have a rule for the other parts of the site ?

Reply:
We use bold to emphasize the definition. In the rest will be node/breaker.

Reply:
@annetill: No rules for the moment. I would simply use Node/Breaker without any specific style (Bus/Breaker and Bus/Branch). I'm not sure italic is really readable, specially on small devices, and I'm not sure to know what I should put in italic, and what I should not
@marqueslanauja: I would also replace `iIDM` by `IIDM` to use the same form everywhere. It's easier to write and read :)

The iTesla website is down, but could we find a link that describe the `European Union 7th Framework Program`

Reply:
The link in the CORDIS system should be available for a long time: https://cordis.europa.eu/project/id/283012

Reply:
iIDM replaced. Link added.

Reply:
Can you please remove the use of specific style as there is not rule and as we want the web site to be homogeneous ? And can you remplace `iIDM` by `IIDM` ? Thanks !